posts/2020-11-20-JNDI-gRPC.adoc

```diff
@@ -0,0 +1,361 @@
+---
+layout: post
+title: "Support for Custom JNDI for Enterprise Beans and gRPC now in GA in Open Liberty 20.0.0.12"
```

Comment:
`now in GA` probably should be just `in`.  The other announcement blog posts do not have `now` or `GA`

Reply:
Done